src/Sylius/Bundle/GridBundle/Tests/DependencyInjection/Compiler/RegisterDriversPassTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\GridBundle\Tests\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\Container;
+use Sylius\Bundle\GridBundle\DependencyInjection\Compiler\RegisterDriversPass;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+class RegisterDriversPassTest extends \PHPUnit_Framework_TestCase
```

Comment:
tbh, I used to use that library myself but over time I (personally) found that it was just obfuscating the tests. imo it's not worth abstracting.